config/supervisord.conf

```diff
@@ -5,6 +5,7 @@ file=/var/run/supervisor.sock
 pidfile=/var/run/supervisor.pid
 logfile=/var/log/supervisor.log
 logfile_backups=1
+environment=DEVELOPMENT=True
```

Comment:
meh, no worries, the biggest thing is that users of this repo, can do vagrant or docker-compose up, and be good to go.